http/src/main/resources/jupyter/init-actions.sh

```diff
@@ -92,6 +92,8 @@ if [ ! -z ${SERVICE_ACCOUNT_CREDENTIALS} ] ; then
   export GOOGLE_APPLICATION_CREDENTIALS=/etc/${SERVICE_ACCOUNT_CREDENTIALS}
 fi
 
+gsutil cp 'custom-env-vars.json' /etc
```

Comment:
Think this needs to be a bucket path. Also it looks like `jupyter-docker-compose.yaml` is expecting a file named `/etc/custom_env_vars.env`, not `custom-env-vars.json`.

Reply:
Will unify filenames.

Using a bucket path would make more sense. I'm having trouble figuring out whether the bucket we upload things to in `initializeBucketOptions` is passed to this script and if so how.

Reply:
There's a few things to do. You can probably use `jupyterNotebookConfigUri` as a model.

1. Add a `customEnvVarsUri` field to [ClusterResourcesConfig](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/ClusterResourcesConfig.scala)
2. Add a `customEnvVars` field to [ClusterInitValues](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala#L271)
3. In `ClusterInitValues.apply`, populate it like [this](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala#L341)
4. In [LeonardoService.initializeBucketObjects](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala#L922), build the content and upload to GCS by adding a call like [this](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala#L971).
5. In [init-actions.sh](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/jupyter/init-actions.sh), add a line like [this](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/jupyter/init-actions.sh#L131)
6. In [init-actions.sh](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/jupyter/init-actions.sh), add a `gsutil cp` like [this](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/jupyter/init-actions.sh#L239-L242)
7. Set the `env_file` in jupyter-docker-compose as you have currently

Reply:
For #5, what actually populates the values on the RHS?